### DISCUSSION TITLE
Asesoria

### DISCUSSION THREAD

**Discussion by eduardoGalvanRmz** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#124] [Q&A]
Buenas noches compañxs de tripulación. Tengo una pregunta para a quellos que tienen oportunidad de ver los live en tiepo y forma.
Por mi trabajo solo he tenido tiempo de ver el live del lunes, aun no he visto el de ayer y hoy, mi pregunta es: ¿El MC ha dado alguna explicación de puntualmente explicita sobre como se entregará la Misión 1? me quede en que trabajaremos con los live que se llevaron acabo la primera semana y con nuestro desempeño autodidacta.
¿Las cosas siguen igual?

Agradeceré enormemente una resúesta, gracias. 👾

**Comment by AngelCruzO** [comment]
En el live del lunes, dijo todo eso

**Comment by carlos-gmx3** [comment]
Todo se sube a un repositorio en github e igual que las katas de Python vas a llenar un formulario con el link al repositorio de las practicas de la semana, el MC no dió hasta el 4 de Marzo para entregar pero recomienda no atrasarse.
